POSEIDON_yml_fields.tsv

```diff
@@ -14,7 +14,8 @@ genoFileChkSum	1	genotypeData	md5 checksum of the genoFile	String		FALSE	TRUE
 snpFile	1	genotypeData	relative path to snpFile	String	Path	TRUE	TRUE
 snpFileChkSum	1	genotypeData	md5 checksum of the snpFile	String		FALSE	TRUE
 indFile	1	genotypeData	relative path to indFile	String	Path	TRUE	TRUE
-indFileChkSum	1	genotypeData	md5 checksum of the indFile	String		FALSE	TRUE
+indFileChSum	1	genotypeData	md5 checksum of the indFile	String		FALSE	TRUE
```

Comment:
`indFileChSum` should be changed back to `indFileChkSum`. Typo?

Reply:
Oops, yes, of course. No idea how that happened.

Reply:
Done.